src/app/pages/OnDemandTvPage/VideoPlayer/index.jsx

```diff
@@ -0,0 +1,123 @@
+import React, { useContext } from 'react';
+import { useLocation } from 'react-router-dom';
+import styled from 'styled-components';
+import {
+  GEL_SPACING,
+  GEL_SPACING_DBL,
+  GEL_SPACING_QUAD,
+} from '@bbc/gel-foundations/spacings';
+import { GEL_GROUP_2_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+import { string, bool } from 'prop-types';
+import {
+  CanonicalMediaPlayer,
+  AmpMediaPlayer,
+  MediaMessage,
+} from '@bbc/psammead-media-player';
+import pathOr from 'ramda/src/pathOr';
+import { RequestContext } from '#contexts/RequestContext';
+import { ServiceContext } from '#contexts/ServiceContext';
+import getEmbedUrl from '#lib/utilities/getEmbedUrl';
+import getPlaceholderImageUrl from '../../../routes/utils/getPlaceholderImageUrl';
+
+const VideoPlayerWrapper = styled.div`
+  width: calc(100% + ${GEL_SPACING_DBL});
+  margin: 0 -${GEL_SPACING};
+  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
+    width: calc(100% + ${GEL_SPACING_QUAD});
+    margin: 0 -${GEL_SPACING_DBL};
+  }
+`;
```

Comment:
For a separate issue - the `CpsAssetMediaPlayer` that we use on MAPs ought to have similar styling. It's meant to be full-width on mobile but isn't currently.

Reply:
Please create a bug issue for the board.

Reply:
👍 have done